policy-utils/src/principal.rs

```diff
@@ -31,6 +32,8 @@ pub enum Principal {
     Participant(u64),
     /// Program in Veracruz, indentified by the program file name.
     Program(String),
+    /// Pipeline in Veracruz, indentified by the pipeline name.
+    Pipeline(String),
```

Comment:
So the `Pipeline` operates as it's own principal?

Reply:
Yes. It is to define a separate right/permission for the entire pipeline. It will ignore programs' individual rights in the pipeline.

Further design is considered, for example, an intersection of pipeline and individual rights.